service/routes.py

```diff
@@ -93,3 +93,23 @@ def index():
 ######################################################################
 
 # Todo: Place your REST API code here ...
+
+
+######################################################################
+# DELETE A PET
+######################################################################
+@app.route("/payment-methods/<int:payment_id>", methods=["DELETE"])
```

Comment:
i think it should be `"payment-method/<int:id>"` as per REST rules; we are deleting a single payment method not some list hence singular noun

also i suggest `id` instead of `payment_id`, as the `id` is clear in the context of this function, and `payment_id` refers to some payment rather **payment method**